apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Snippets/JobCreationOptionsSnippets.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using static Google.Apis.Bigquery.v2.JobsResource.ListRequest;
+
+namespace Google.Cloud.BigQuery.V2.Snippets
+{
+    [SnippetOutputCollector]
+    [Collection(nameof(BigQuerySnippetFixture))]
+    public class JobCreationOptionsSnippets
+    {
+        private readonly BigQuerySnippetFixture _fixture;
+
+        public JobCreationOptionsSnippets(BigQuerySnippetFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void ListJobs_FilterByLabels()
+        {
+            string bucketName = _fixture.StorageBucketName;
+            string objectName = _fixture.GenerateStorageObjectName();
+
+            string projectId = _fixture.ProjectId;
+            string datasetId = _fixture.GameDatasetId;
+            string tableId = _fixture.HistoryTableId;
+
+            // Snippet: Labels
+            IDictionary<string, string> labels = new Dictionary<string, string>()
+            {
+                {"label-key", "label-value" }
+            };
+
+            BigQueryClient client = BigQueryClient.Create(projectId);
+            BigQueryTable table = client.GetTable(projectId, datasetId, tableId);
+            string destinationUri = $"gs://{bucketName}/{objectName}";
+
+            // Just a couple examples of jobs marked with labels:
+            // (These jobs will most certainly be created somewhere else.)
```

Comment:
Not sure what you mean by this parenthesized line.
(We haven't generally added snippets for options, but there's certainly no harm in doing so.)

Reply:
With the parenthesized line I meant that you wouldn't create jobs "just for the sake of listing them", that is, that those jobs would have been created elsewhere as part of normal execution of the application.

My original intention was not to add the snippet to the options, I wanted to add it as a second sample for ListJobs(*) (list jobs and filter by labels) but couldn't find a way to do so. Don't know yet if it's possible to have two snippets per method signature, don't know if the second snippet is not generated at all or if it is just not included as part of the class docs, and if this is by design or not, wanted to ask you about it after looking at the code a little.

Reply:
My original intention was not to add the snippet to the options, I wanted to add it as a second sample for ListJobs(*) (list jobs and filter by labels) but couldn't find a way to do so. Don't know yet if it's possible to have two snippets per method signature, don't know if the second snippet is not generated at all or if it is just not included as part of the class docs, and if this is by design or not, wanted to ask you about it after looking at the code a little.